.travis.yml

```diff
@@ -0,0 +1,43 @@
+language: python
+
+python:
+    - "2.7"
```

Comment:
Python 3 would be preferable if there's nothing stopping us from using it.

Reply:
The only thing using python here is dx-toolkit, which sadly only works with python 2.7.